src/organisms/Narrative.tsx

```diff
@@ -0,0 +1,100 @@
+import * as React from 'react';
```

Comment:
I didn't follow atomic design format for this at all because the Narrative concept feels very tied to just Explore page right now. If this concept seems like it would be applied to other pages it might make more sense to abstract out the stuff in here into a more atomic-oriented structure. But for now it is a special snowflake organism